src/ExtranetBundle/LoginByTransferToken/LoginController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace ChameleonSystem\ExtranetBundle\LoginByTransferToken;
+
+use ChameleonSystem\CoreBundle\Service\PageServiceInterface;
+use ChameleonSystem\ExtranetBundle\Interfaces\ExtranetUserProviderInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+
+class LoginController
```

Comment:
LoginByTokenController ?

Otherwise it suggests that every login is handled here.

Reply:
Fair.

:heavy_check_mark: Renamed to `LoginByTokenController`